packages/core/types/stitches.d.ts

```diff
@@ -204,7 +204,7 @@ type ThemeTokens<Values, Prefix> = {
 	[Scale in keyof Values]: {
 		[Token in keyof Values[Scale]]: ThemeUtil.Token<
 			Extract<Token, number | string>,
-			Values[Scale][Token],
+			Values[Scale][Token] extends string | number ? Values[Scale][Token] : string,
```

Comment:
I think replacing this line with 
```
			Values[Scale][Token] & (string | number),
```
should be simpler and would produce the same results since we essentially just want to remove the symbol type and doing it via intersection would produce a more readable results

Reply:
Good call, @hadihallak. I tested it locally, and it seems to be working just fine as well. I'll make the change in both files.